dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -145,6 +151,38 @@ public ResponseEntityView<String> delete(@Context final HttpServletRequest reque
         return new ResponseEntityView<>("Experiment deleted");
     }
 
+    /**
+     * Returns a list of {@link Experiment}s optionally filtered by pageId, name or status.
+     */
+    @GET
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public ResponseEntityExperimentView list(final @QueryParam("pageId") String pageId,
```

Comment:
I think we can improve this doc, for example how it is the output format of this end point?

Reply:
How would be an example of this? not sure what you want here @freddyucv